.travis.yml

```diff
@@ -18,19 +18,23 @@ install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
+  - conda config --add channels conda-forge
+  - conda config --add channels skywalker-tag
+  - conda config --add channels lightsource2-tag
+  - conda install conda-build
+  - conda update -q conda conda-build
```

Comment:
This is a nicer way to do this. Order added is the order the opposite order they are searched?

Reply:
I'll have to double-check whether they add to the front or the back of the queue.